### ISSUE TITLE
POC Overview usage bar charts

### ISSUE THREAD

**Issue by dlabrecq** [project-koku/koku-ui#1086]
Replace "Compute instance types" and "storage services" charts with bar charts. This would apply to Azure, Aws, and Ocp. Confirm with @nlcwong that the 30-60 data points looks ok.

**Comment by dlabrecq** [comment]
FYI, @nlcwong @chargio 

I captured some screen shots from a POC that utilizes bar charts. Had to make the bars thin, so it looks ok with 2 sets of data. It's too crowded with 4 sets of data, even with 2 cards per row.

OCP on AWS tab, 3 cards per row
![Screen Shot 2019-11-13 at 4 04 51 PM](https://user-images.githubusercontent.com/17481322/68809045-08dfab00-0639-11ea-9914-3a23a0e9846e.png)

OCP on AWS tab, 2 cards per row
![Screen Shot 2019-11-13 at 4 06 23 PM](https://user-images.githubusercontent.com/17481322/68809094-29a80080-0639-11ea-9e45-554a348a2794.png)

**Comment by dlabrecq** [comment]
This is what 2 cards per row look like when the window is resized smaller. This uses a multi-color theme, so the bars are more visible.

Note that the bars overlap because there simply is not enough space.

![Screen Shot 2019-11-13 at 4 22 54 PM](https://user-images.githubusercontent.com/17481322/68809194-683dbb00-0639-11ea-9357-a5d242f83b36.png)

**Comment by dlabrecq** [comment]
This is what 2 cards per row look like when the window is resized smaller. This uses a multi-color theme, so the bars are more visible.

Note that the bars overlap because there simply is not enough space.

![Screen Shot 2019-11-13 at 4 19 47 PM](https://user-images.githubusercontent.com/17481322/68809275-91f6e200-0639-11ea-9b1f-6fb5522ec7b5.png)

**Comment by dlabrecq** [comment]
This is an example of the OCP on AWS details historical charts. These charts show 6 sets of data.

![Screen Shot 2019-11-13 at 5 06 58 PM](https://user-images.githubusercontent.com/17481322/68809409-e306d600-0639-11ea-9eb7-9910a216b254.png)

Same historical charts, but without limits data

![Screen Shot 2019-11-13 at 5 08 59 PM](https://user-images.githubusercontent.com/17481322/68809444-f023c500-0639-11ea-99b1-c75edda8b56a.png)

**Comment by chargio** [comment]
I agree that in this case it could be beneficial to show the two months one after the other

**Comment by chargio** [comment]
Meaning: not overlapping

**Comment by nlcwong** [comment]
@chargio doesn't matter if we show it "overlapping" or have it nov 1 - dec 31, if you look at it either way, having that many bars w/in the viewport that we have it is too much. the width is just not big enough for the amount of data points we want to show on the graph.  So, my question to you is if there is a real need to show usage vs. requests?  Can we just have a graph to just show usage and historical usage and another graph (not on the same graph) show requests and historical requests?
